evap/static/scss/components/_modal.scss

```
@@ -1,3 +1,9 @@
 .modal-xl {
     max-width: 1120px;
 }
+
+.modal-submit-grp {
```

```suggestion
.modal-submit-group {
```
please use a speaking name for this class